.editorconfig

```diff
@@ -10,7 +10,7 @@ insert_final_newline = true
 trim_trailing_whitespace = true
 indent_style = tab
 
-[{package.json,*.yml}]
+[{*.json,*.yml,*.yaml,.stylelintrc,.jshintrc,.eslintrc,.jscsrc}]
```

Comment:
This change is no longer needed, all `.json` files should now use tabs for indentation

See https://core.trac.wordpress.org/ticket/40946